### DISCUSSION TITLE
Implement native blur view on Android

### DISCUSSION THREAD

**Discussion by OmarBasem** [status-im/status-legacy#16684] [General]
Discussion: https://github.com/status-im/status-mobile/discussions/15440

Android Window Blur: https://source.android.com/docs/core/display/window-blurs

**Comment by briansztamfater** [comment]
I think we need blurs at the Android View level, not Android Window. Good news is that Android 12 introduced native APIs to apply blur to views also: https://developer.android.com/reference/android/graphics/RenderEffect#createBlurEffect(float,%20float,%20android.graphics.Shader.TileMode)

Let me know if you need any help trying them.

**Comment by OmarBasem** [comment]
Hi @briansztamfater, I currently got busy with other issues. I can look into it next week. If you can look into it this week, then feel free to assign yourself

**Comment by briansztamfater** [comment]
Trying a different approach with WebView and CSS blurs here #15953
